packages/zoltan2/test/driver/driverinputs/kuberry/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Test problem that exposes a memory leak in MJ when num_global_parts==1
+TRIBITS_ADD_TEST(
+    test_driver
+    NAME kuberry
+    DIRECTORY ../..
+    NUM_MPI_PROCS 4
```

Comment:
No need to specify num_mpi_procs; OK to run in serial, too.

Reply:
Removed